packages/@coorpacademy-components/src/atom/radio-with-title/style.css

```diff
@@ -0,0 +1,49 @@
+@value colors: "../../variables/colors.css";
+@value white from colors;
+@value dark from colors;
+@value cm_grey_200 from colors;
+@value cm_grey_100 from colors;
+@value black from colors;
+@value cm_primary_blue from colors;
+
+.container {
+  display: flex;
+  align-items: center;
+  align-content: flex-start;
+}
+
+.container label {
+  display: flex;
+}
+
+.radio {
+  appearance: none;
+  width: 24px;
+  height: 24px;
+  border-radius: 27px;
+  background: cm_grey_100;
+  border: 1px solid cm_grey_200;
+  margin-right: 16px;
+}
+
+.radio:checked {
+  appearance: none;
+  box-shadow: inset 0 0 0 0.4em cm_primary_blue;
```

Comment:
0.5em looks match more with the Figma, no?

This is with 0.4em
<img width="391" alt="Capture d’écran 2023-06-26 à 11 04 16" src="https://github.com/CoorpAcademy/components/assets/7602475/b0b15a73-695a-463c-b029-83f22eafda44">

This is with 0.5em
<img width="360" alt="Capture d’écran 2023-06-26 à 11 04 05" src="https://github.com/CoorpAcademy/components/assets/7602475/af3f2c30-e421-46e4-8e6f-a8750f4f7813">

This is the Figma
![Capture d’écran 2023-06-26 à 11 04 26](https://github.com/CoorpAcademy/components/assets/7602475/a4244694-4819-4676-9668-72938815add1)